dev/breeze/src/airflow_breeze/utils/md5_build_check.py

```diff
@@ -102,11 +103,15 @@ def calculate_md5_checksum_for_files(
         if modified_provider_yaml_files:
             get_console().print(
                 "[info]Attempting to generate provider dependencies. "
-                "Provider yaml files changed since last check:[/]"
-            )
-            get_console().print(
-                [os.fspath(file.relative_to(AIRFLOW_SOURCES_ROOT)) for file in modified_provider_yaml_files]
+                f"{len(modified_provider_yaml_files)} provider.yaml file(s) changed since last check:[/]"
```

Comment:
Do we need the `:[/]` here anymore at the end of print line?

Reply:
Nope.

Reply:
Fixed.